lib/atom-alias-command.coffee

```diff
@@ -1,10 +1,15 @@
 {CompositeDisposable} = require 'atom'
 
+typeIsArray = Array.isArray || ( value ) -> return {}.toString.call( value ) is '[object Array]'
+
 module.exports = aliasCommand = (as, {orig, scope}) ->
   scope ?= 'atom-workspace'
   aliasCommand.subscriptions.add(
     atom.commands.add scope, as, (event) ->
-      atom.commands.dispatch event.target, orig
+      if typeIsArray orig
+        atom.commands.dispatch event.target, o for o in orig
```

Comment:
Found these related discussions:
https://github.com/atom/atom/issues/2670#issuecomment-46664964
https://github.com/atom/atom/commit/9aee7d4b381664cb4876f620a97049738410f728

Sounds like Atom core's stance is that composing commands at the `dispatch` level is ambiguous (series or parallel? what if one fails?), and that the ambiguity should be resolved by defining a new command that does the desired composition at the JS API level.  

Seems to me like a reasonable stance to support with this plugin. Principled objection?